resource-managers/kubernetes/core/pom.xml

```diff
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.spark</groupId>
     <artifactId>spark-parent_2.11</artifactId>
-    <version>2.2.0-SNAPSHOT</version>
+    <version>2.1.0</version>
```

Comment:
The same version string as the image makes sense?
`2.1.0-k8s-support-0.1.0-alpha.1`?